app/assets/javascripts/discourse/app/models/post-stream.js

```diff
@@ -402,7 +401,10 @@ export default RestModel.extend({
             this._initUserModels(p);
             const stored = this.storePost(p);
             if (!currentPosts.includes(stored)) {
-              currentPosts.insertAt(postIdx++, stored);
+              const insertAtIndex = postIdx++;
+              this.postsWithPlaceholders.insertPost(insertAtIndex, () => {
```

Comment:
what's `insertPost`?

Reply:
Good question... it's supposed to be a method I added to posts-with-placeholders... but it looks like it must've got lost at some point over the months of rebasing 😓 

Will figure out what happened, and also make sure we have some kind of test for this 'gap expand' logic.

Thanks for catching this! 🙏